dispatch.yaml

```diff
@@ -1,3 +1,16 @@
 dispatch:
-- url: "*/tasks/*"
+
+# This will never match, because /_ah/ requests are not routed.
+# https://cloud.google.com/appengine/docs/standard/python/reference/dispatch-yaml
+# This is why app_py must remain the default service must remain in the
```

Comment:
Sorry, I had a text editing error in that comment.  I have rewritten the comment to fix the grammar and to clarify that requests to /_ah/* are not routed by dispatch.yaml (but they can match handler lines within a service yaml file).

Regarding that wpt.fyi yaml file, I'd guess that the liveness check is not routed through dispatch.yaml either.  But instead of all going to the default service, the liveness checks are routed to each instance of each service that has a liveness check configured.